src/content/translations/vi/dapps/index.md

```diff
@@ -0,0 +1,39 @@
+---
```

Comment:
As mentioned in Step 4 of [the issue](https://github.com/ethereum/ethereum-org-website/issues/1331), we should replace the `meta` block with a `description` property.

So the current frontmatter:
```
---
title: Sử dụng ứng dụng Ethereum (ứng dụng phi tập trung)
meta:
  - name: Mô tả
    content: Thông tin cơ bản bạn cần để bắt đầu sử dụng Ethereum.
  - property: title
    content: Sử dụng ứng dụng Ethereum (ứng dụng phi tập trung) | Ethereum.org
  - property: description
    content: Thông tin cơ bản bạn cần để bắt đầu sử dụng Ethereum.
lang: vi
---
```

Should be:
```
---
title: Sử dụng ứng dụng Ethereum (ứng dụng phi tập trung)
description: Thông tin cơ bản bạn cần để bắt đầu sử dụng Ethereum.
lang: vi
---

Reply:
This need to be updated on every markdown page.